toonz/sources/CMakeLists.txt

```diff
@@ -198,13 +219,65 @@ else()
 
     find_package(LZO REQUIRED)
     message("LZO:" ${LZO_INCLUDE_DIR})
+
+elseif(UNIX)
+    find_library(GL_LIB GL)
+    find_library(GLU_LIB GLU)
+    find_library(GLUT_LIB glut)
+    find_library(Z_LIB z)
+
+    find_package(TIFF REQUIRED)
+    set(TIFF_LIB ${TIFF_LIBRARY})
+    # the libraries have not .pc nor preferred Find*.cmake. use custom modules.
+    # find_package(PNG REQUIRED)
+    set(PNG_LIB ${SDKROOT}/libpng-1.6.21/.libs/libpng16.so)
```

Comment:
Many distros ship current versions of libpng, can this be changed into a fallback only when system libpng is not found?